cylc/flow/main_loop/log_memory.py

```diff
@@ -42,6 +42,9 @@
 
    Set a sensible interval before running suites.
 
+If ``matplotlib`` is installed this plugin will plot results as a PDF in
+the run directory when the suite is shut down (cleanly).
```

Comment:
That cleanly note will be helpful too. I stopped the suite a few times with CTRL+C and went looking for the plot, then tried `cylc stop five` and that worked.

Reply:
Should add a suite abort hook too at some point.